### DISCUSSION TITLE
Changing CSS Folder location and quasar.variable.scss

### DISCUSSION THREAD

**Discussion by numero14-dev** [quasarframework/quasar#9969] [General - Components / Directives / etc]
Hi
I'm looking for a way for changing the CSS folder and Quasar.variable.scss file location. but when I changing the folder location I got an error that says quasar.variable.scss file not found in src/css folder. 
I searched but did not find any solution.
is there any way to change the CSS folder and quasar.variable.scss file location? 

**Ex:** Transferring CSS folder location with All files inside it (quasar.variable.scss & app.scss) to Assets folder.

Thanks

**Comment by justinbeaty** [comment]
@numero14-dev you probably could with a rather complicated `extendWebpack` or `chainWebpack` rule, but I must ask, why in particular do you want to do this?
